projects/angular/src/emphasis/_badges.clarity.scss

```
@@ -6,43 +6,25 @@
 
 @use 'sass:meta';
 @use '../utils/mixins';
-@use '../utils/variables/variables.global';
-@use 'variables.badges' as badges-variables;
-@use 'variables.label' as label-variables;
+@use '@cds/core/tokens/tokens.scss';
 @include meta.load-css('properties.badges');
 
+@function getBadgeCssVarName($type: gray, $prop: color) {
+  @return #{'clr-badge-' + $type + '-' + $prop};
+}
```

Add dashes in the generation

```suggestion
@function getBadgeCssVarName($type: gray, $prop: color) {
  @return #{'--clr-badge-' + $type + '-' + $prop};
}
```

or convert it to mixin
```suggestion
@mixin badge-css-var($style: gray, $type: gray, $prop: color) {
  #{$style}: var(#{'--clr-badge-' + $type + '-' + $prop});
}
```

Both approaches are valid.